lib/sotest.nix

```diff
@@ -94,4 +97,60 @@ rec {
       exec = builtins.concatStringsSep " " ([ kernel ] ++ kernelParams);
       load = [ initrd ];
     };
+
+  # Given a test run attribute set, this function is used to build a bundle consisting of a project
```

Comment:
For n00b users like me, it would be good to have a quick pointer to how a `testRunConfig` looks like as a refresher. Or an example would also help.